public/index.html

```diff
@@ -1,41 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
For now, undo all changes to this file, do not edit it in the future, until we decide that we must do it